### PR TITLE
Do not try to load data if we are already doing it, fixes #910

### LINKED ISSUE #910: `mv-login` fired during the data loading breaks the autosave feature
Reduced testcase: https://codepen.io/dmitrysharabin/pen/ZExZaOJ

### PR DESCRIPTION
**Pull request by DmitrySharabin**

### CODE REVIEW

**Review by LeaVerou — CHANGES_REQUESTED** [review]
Doesn't `inProgress` contain UI text for *any* kind of in-progress event, and not just data loading? Perhaps there's a conditional that will *only* match when data is being loaded?

**Comment by DmitrySharabin** [comment]
> Doesn't `inProgress` contain UI text for _any_ kind of in-progress event, and not just data loading? Perhaps there's a conditional that will _only_ match when data is being loaded?

You are absolutely right; we have such a conditional. Previously, I thought there was no need to be as precise as to handle the loading in-progress event only. Now, I believe that we should. I have committed the fix. Thank you!

File: src/mavo.js
Comment (LeaVerou) [inline src/mavo.js:138]: This is not localized?

Comment (DmitrySharabin) [reply src/mavo.js:138]: For now, the only thing that is localized (internally) is `Uploading` (https://github.com/search?q=repo:mavoweb/mavo%20this.inProgress&type=code), but not everywhere. However, all the in-progress events are localized and are present in `locale.en.js`. Should I submit a separate PR to localize the rest of the events? Or we can have everything done in this one.

Comment (DmitrySharabin) [reply src/mavo.js:138]: Btw, we already discussed in-progress events localization here: https://github.com/mavoweb/mavo/pull/799#issuecomment-983952916

And there, we have some cases where localization might be undesirable.

Comment (DmitrySharabin) [reply src/mavo.js:138]: One more thing: `inProgress` is an internal thing. When we show something to the user, we use the localized strings. So I believe we don't need to localize it in other places.

https://github.com/mavoweb/mavo/blob/286de53ec3276a2828f2488d1aaaa5b6170bc631/src/mavo.js#L926-L930

Comment (LeaVerou) [reply src/mavo.js:138]: We should probably lowercase the values (e.g. `"loading"` instead of `"Loading"` to make it clear this is an enum, and not user facing text), then we'd use these as localization keys for the user facing text.

Comment (LeaVerou) [reply src/mavo.js:138]: And yes, it looks like this should be a separate PR, that is merged before this one.

Comment (DmitrySharabin) [reply src/mavo.js:138]: We'll do it if you don't mind. :)

Comment (DmitrySharabin) [reply src/mavo.js]: Done!

**Review by LeaVerou — APPROVED** [review]
LGTM, approved, feel free to merge once the other one is merged

**Pull request merged**
